.github/workflows/linter.yml

```diff
@@ -49,10 +49,10 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: "16"
-      - run: npm install --production
+      - run: npm install
```

Comment:
Do we want to install everything in dev, or just the linter?  We might want to move that package over instead.

Reply:
So this installation is actually for the build process that happens below. At least that's what I think. And we need all that stuff for the CSS to build.